eth_account/hdaccount.py

```diff
@@ -0,0 +1,154 @@
+import os
+import hmac
+import hashlib
+import struct
+import codecs
+import ecdsa
+
+from hashlib import sha256
+from ecdsa.curves import SECP256k1
+from ecdsa.ecdsa import int_to_string, string_to_int
+
+MIN_ENTROPY_LEN = 128        # bits
+BIP32_HARDEN    = 0x80000000 # choose from hardened set of child keys
+CURVE_GEN       = ecdsa.ecdsa.generator_secp256k1
+CURVE_ORDER     = CURVE_GEN.order()
+EX_MAIN_PRIVATE = [ codecs.decode('0488ade4', 'hex') ] # Version strings for mainnet extended private keys
```

Comment:
Can you link to a reference for where this magic number comes from?

Reply:
These numbers come from the bip https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki
I don't know how to specify the line in that file so please go there and search for the numbers >.<

Reply:
https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#serialization-format <- that link is sufficient, and should go in a source comment